backend/sirius-web-diagrams/src/main/java/org/eclipse/sirius/web/diagrams/components/NodeComponent.java

```diff
@@ -78,6 +82,17 @@ public Element render() {
         return new Fragment(fragmentProps);
     }
 
+    private boolean viewCreationRequestedFor(Object semanticElement) {
+        NodeDescription nodeDescription = this.props.getNodeDescription();
+        VariableManager variableManager = this.props.getVariableManager();
+        Optional<Diagram> optionalDiagramContainer = variableManager.get("ContainerView", Diagram.class); //$NON-NLS-1$
+        Optional<Node> optionalNodeContainer = variableManager.get("ContainerView", Node.class); //$NON-NLS-1$
```

Comment:
No need to use the variable manager to give the id of the container view to its child, use props. VariableManager are given to the handlers of the specifier. Adding something to the variable manager is creating a new public API for the specifier. They don't need it so add `containerViewId` to NodeComponentPops directly (since you have `targetObjectId` and `description.getId()` already in this class.